shared/login/register/username-or-email/index.js

```diff
@@ -4,8 +4,32 @@ import {connect} from 'react-redux'
 import Render from './index.render'
 import type {Props} from './index.render'
 
+let setup = false
+
+let count = 0
+const setupThrash = (dispatch) => {
```

Comment:
this is just a helper to show the bad state. when you go into the provision flow it'll keep swapping the busy state which should only affect the button. with master's code it'd refresh the entire tree (you'll lose focus and anything you've typed into the username field)